src/config/ConfigScope.ts

```diff
@@ -100,6 +100,15 @@ export class ConfigScope {
     )
   }
 
+  getOptionalOneOf(param: string, defaultValue: string, supportedValues: string[]): string {
```

Comment:
<T extends string> would provide better type narrowing here. Check how getMandatoryOneOf is implemented.

Reply:
I think it is addressed already, thanks Igor 🙇